VK_V1.0/VK_V1.0/Servics/NetworkService.swift

```diff
@@ -0,0 +1,93 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Глобальный список типов данных
+enum TypeMethod {
+    case friends
+    case photos
+    case myGroups
+    case availableGroups
+}
+
+/// Загрузка данных с сервера
+final class NetworkService {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let friendMethod = "/method/friends.get"
+        static let photosMethod = "/method/photos.getAll"
+        static let myGroupsMethod = "/method/groups.get"
+        static let availableGroupsMethod = "/method/groups.search"
+        static let userIdName = "user_id"
+        static let versionName = "v"
+        static let versionValue = "5.131"
+        static let fieldsName = "fields"
+        static let extendedName = "extended"
+        static let extendedValue = "1"
+        static let searchTextName = "q"
+        static let fieldsValue = "first_name, photo_50"
+        static let ownerIdName = "owner_id"
+        static let countGroupsName = "count"
+        static let countGroupsValue = "40"
+    }
+
+    // MARK: - Public methods
+
+    func loadData(data: TypeMethod, for ownerId: String?, searchText: String?) {
```

Comment:
```suggestion
    func loadData(data: TypeMethod, for ownerId: String?, searchText: String?) {
```
тут кейсы не подойдут так как методов может быть 200 это не читаемо раздели на маленькие методы